vnext/Scripts/IntegrationTests.ps1

```diff
@@ -1,97 +1,94 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
-param(
-	[switch] $NoSetup,
-	[switch] $Run,
-	[switch] $Cleanup,
+#
+# IntegrationTests.ps1
+#
+param (
+	[switch] $NoServers,
```

Comment:
>[switch] $NoServers [](start = 1, length = 19)

Add documentation for the params ([parameter(HelpMessage="help text for param")])?